types/node/scripts/generate-docs/html-doc-processing.ts

```diff
@@ -100,6 +100,13 @@ export function fixupLocalLinks(text: string, moduleName: string): string {
 
 export function fixupHtmlDocs(input: string, context: DocContext): string {
     // make sure code highlighting works
+
+    // This is not fast but it will do, also he is definitetely coming for us now.
```

Comment:
Typo: "definitely"

Reply:
One would think I'd know how to spell this by now...